### ISSUE TITLE
Cannot start buildbuddy with v1.0.4

### ISSUE THREAD

**Issue by prestonvanloon** [buildbuddy-io/buildbuddy#25]
```
2020/05/17 18:55:36 Reading buildbuddy config from 'config/buildbuddy.onprem.yaml'
2020/05/17 18:55:36 Error loading config from file: Config file config/buildbuddy.onprem.yaml not found
```

Args:
- --config_file=config/buildbuddy.onprem.yaml

This worked with the v1.0.0 image.

**Comment by prestonvanloon** [comment]
Changing this to `config/buildbuddy.release.yaml` worked.

**Comment by siggisim** [comment]
Hey @prestonvanloon - sorry about that. We updated the configuration files a bit and now only buildbuddy.release.yaml is packaged with the Docker image we ship.

We updated our sample deployment script with the changes:
https://github.com/buildbuddy-io/buildbuddy/blob/0bdbeff1210267541bbb6198681ba6a296c6b30c/deployment/buildbuddy-app.onprem.yaml#L29

And added some documentation on how to supply a custom config file to your Docker image:
https://github.com/buildbuddy-io/buildbuddy/blob/0bdbeff1210267541bbb6198681ba6a296c6b30c/deployment/buildbuddy-app.onprem.yaml#L29

Sorry for the confusion, we'll try to make changes like this more obvious in future release notes. Thanks for bring it to our attention!
